packages/manager/apps/pci-instances/src/components/priceLabel/PriceLabel.component.tsx

```diff
@@ -3,9 +3,9 @@ import { OsdsText } from '@ovhcloud/ods-components/react';
 import { ODS_TEXT_SIZE, ODS_TEXT_LEVEL } from '@ovhcloud/ods-components';
 import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
 import { useCatalogPrice } from '@ovh-ux/manager-react-components';
-import { TInstancePrice } from '@/types/instance/common.type';
+import { TInstancePrice } from '@/types/instance/entity.type';
 
-const PriceLabel: FC<Omit<TInstancePrice, 'status'>> = ({ value, type }) => {
+const PriceLabel: FC<Omit<TInstancePrice, 'label'>> = ({ value, type }) => {
```

Comment:
Isn't it a little strange to have a `PriceLabel` component which omit the `label` prop?